source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1954,6 +1954,14 @@ def event_alert(self):
 				speech.speakObject(child,reason=controlTypes.REASON_FOCUS)
 			child=child.simpleNext
 
+
+class UIItem(IAccessible):
+	"""List items in Windows Explorer repeate the name as the value"""

```

Comment:
Typo: 'repeate' should be "repeat"
```suggestion
	"""List items in Windows Explorer repeat the name as the value"""

```

Reply:
@feerrenrut  wrote:
> 
> Typo: 'repeate' should be "repeat"

It was bad copy-paste from UIA module. Fixed now.

Regarding behavior under different versions of Windows - I've tested on  Win 7, 8.1 and Win 10 1511 and 1909 and on all these versions  behavior was the same. It is theoretically possible that on one of  not tested versions it works differently but in my view this is quite unlikely.